.changeset/ninety-candles-perform.md

```diff
@@ -0,0 +1,7 @@
+---
+"@internal/fuel-core": patch
```

Comment:
1. Why remove `@internal-fuel-core`?
1. Why add `crete-fuels`?

Reply:
1. We need to bump it anyway because other apps use it.
2. Good catch! https://github.com/FuelLabs/fuels-ts/pull/2717/commits/3fb86a9012c1432e7d6df663c153a498d3a12388